modules/docker/run.go

```diff
@@ -59,7 +62,7 @@ func Run(t testing.TestingT, image string, options *RunOptions) string {
 
 // RunE runs the 'docker run' command on the given image with the given options and return stdout/stderr, or any error.
 func RunE(t testing.TestingT, image string, options *RunOptions) (string, error) {
-	logger.Logf(t, "Running 'docker run' on image '%s'", image)
+	options.Logger.Logf(t, "Running 'docker run' on image '%s'", image)
```

Comment:
You need to check for `nil` here, as it will default to `nil` unless the user explicitly passes it in. This means that this code, as written, will cause a panic for users updating to this version as opposed to a compile error.

Reply:
As mentioned in https://github.com/gruntwork-io/terratest/pull/510#discussion_r425459142, this is being handled in the Logger type so will mark as resolved.